ElementX/Sources/Screens/Timeline/TimelineViewModelProtocol.swift

```diff
@@ -17,4 +17,7 @@ protocol TimelineViewModelProtocol {
     func process(composerAction: ComposerToolbarViewModelAction)
     /// Updates the timeline to show and highlight the item with the corresponding event ID.
     func focusOnEvent(eventID: String) async
+    
+    /// Handles getting the content to forward an item given its item ID.
+    func getForwardingItem(for itemID: TimelineItemIdentifier) async -> MessageForwardingItem?
```

Comment:
Let's call this `makeForwardingItem`, give it isn't getting it from anywhere, it is actually making one.
```suggestion
    func makeForwardingItem(for itemID: TimelineItemIdentifier) async -> MessageForwardingItem?
```